tests/integration/test_dss.py

```diff
@@ -0,0 +1,42 @@
+import subprocess
+
+
+def test_initialize_creates_dss() -> None:
```

Comment:
these tests do not clean up after themselves, meaning this could happen:

* run tests
  * tests pass (as expected)
* tests create namespace, deployment, etc successfully
* change the code in a way that breaks the tests (for example, edit manifest.yaml and delete the deployment)
* run tests again
  * tests pass (not as expected)

Reply:
Added a fixture that does this, albeit crudely.  Maybe there's a better way we could do in future, but this is a start